src/Kiota.Builder/Refiners/GoRefiner.cs

```diff
@@ -265,6 +266,9 @@ private void RemoveHandlerFromRequestBuilder(CodeElement currentElement)
             "github.com/microsoft/kiota-abstractions-go/serialization", "ParseNode", "Parsable"),
         new (static x => x is CodeClass codeClass && codeClass.IsOfKind(CodeClassKind.Model),
             "github.com/microsoft/kiota-abstractions-go/serialization", "Parsable"),
+        new (static x => x is CodeMethod method && method.IsOfKind(CodeMethodKind.Serializer, CodeMethodKind.Deserializer)
+            && method.Parent is CodeClass codeClass && codeClass.GetPropertiesOfKind(CodePropertyKind.Custom).Where(static x => !x.ExistsInBaseType).Any(),
+            "github.com/microsoft/kiota-abstractions-go", "*utils"),
```

Comment:
Only import class when the serializer is used

Reply:
I'm not sure why we have a pointer symbol in front?

Reply:
Without the pointer symbol the import statement alias is evaluated back to the abstractions hash constant  `conventions.AbstractionsHash`, the pointer symbol in the case forces the import statement to use `util` as an alias.

On this, I had earlier left it to evaluate to the default `conventions.AbstractionsHash` constant since all other references to the abstractions package always have the same import, we might have deal with this issue in future if any more code is added to the models where the import was evaluated as  `conventions.AbstractionsHash`

Reply:
why is the import being evaluated to abstraction hash an issue? the functions declared in the util "file" are part of the abstractions package, it'll be one less import in place where we already import abstractions and you won't have a hard coded constant in the code method writer.

Reply:
resolved